components/compliance-service/dao/pgdb/upgradeDbTest.go

```diff
@@ -8,6 +8,27 @@ type UpgradeDbTest struct {
 	NeedStatus   bool
 }
 
+func (u UpgradeDbTest) GetUpgradeFlags() (map[string]bool, error) {
+	if u.Error {
+		return nil, errors.New("Unable to fetch status from database")
+	}
+
+	flagsMap := make(map[string]bool)
+
+	if u.NeedStatus {
+		flagsMap[ControlIndexFlag] = true
+		flagsMap[CompRunInfoFlag] = true
+		flagsMap[DayLatestFlag] = true
+		return flagsMap, nil
+	}
+	return flagsMap, nil
+}
+
+func (u UpgradeDbTest) UpdateCompRunInfoFlagToTrue() error {
+	//TODO implement me
```

Comment:
Remove TODO statement, panic("implement me") is fine

Reply:
working on code smell